### PR TITLE
WRP-7019: Update package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies to the latest minor version.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated dependencies to the latest minor version.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-7019

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong taeyoung.hong@lge.com

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,15 +44,15 @@
...
     "@babel/eslint-plugin": "^7.19.1",
-    "@babel/plugin-transform-modules-commonjs": "^7.20.11",
+    "@babel/plugin-transform-modules-commonjs": "^7.21.2",
     "@enact/dev-utils": "^5.1.1",
     "@enact/template-sandstone": "^1.5.2",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
     "@testing-library/dom": "^8.20.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
```
Comment (MikyungKim) [inline package.json:54]: These two modules have 14 versions. would you make a ticket?

**Review by MikyungKim — COMMENTED** [review]
Would you please make a ticket for updating the dependency of CLI up to major versions? I'm seeing lots of them.

**Comment by MikyungKim** [comment]
Please resolve the merge conflict.

**Comment by hong6316** [comment]
> Would you please make a ticket for updating the dependency of CLI up to major versions? I'm seeing lots of them.

OK, WRP-11567

> Please resolve the merge conflict. Maybe you need to update your branch with the recent develop.

OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**